App.js

```diff
@@ -1,13 +1,81 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, TextInput, Pressable } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 
-export default function App() {
+import Button from './src/components/Button';
+import Input from './src/components/Input';
+
+
+function Home () {
+  return (
+    <View style={styles.container}>
+      <Text>Please enter credentials</Text>
+      <Input placeholder="Username" />
+      <Input placeholder="Password" secureTextEntry={true} />
+      <Button title="Sign in" />
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+
+function Profile () {
+  return (
+    <View style={styles.container}>
+      <Text>It is profile screen!</Text>
+      <StatusBar style="auto" />
+    </View>
+  )
+}
+
+function List () {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Text>It is list screen!</Text>
       <StatusBar style="auto" />
     </View>
+  )
+}
+
+function Swiper () {
+  return (
+    <View style={styles.container}>
+      <Text>It is swiper screen!</Text>
+      <StatusBar style="auto" />
+    </View>
+  )
+}
+
+const Tab = createBottomTabNavigator();
+
+function App () {
+  return (
+    <NavigationContainer>
+      <Tab.Navigator initialRouteName="Home">
+        <Tab.Screen name="Home" component={Home} options={{
+          tabBarIcon: ({color, size}) => (
+            <MaterialCommunityIcons name="home" color={color} size={size} />
```

Comment:
Your `color` property is the same as you take from the property as default I think. So `color` is probably redundant here.

Reply:
No, when I remove color property the icon become to black color. I want to keep settings the same )